src/Aspire.Hosting/build/Aspire.Hosting.props

```diff
@@ -4,6 +4,7 @@
     <!-- Aspire hosting projects aren't publishable right now until https://github.com/dotnet/aspire/issues/147 is good -->
     <IsPublishable Condition="'$(IsPublishable)' == ''">false</IsPublishable>
     <IsPackable Condition="'$(IsPackable)' == ''">false</IsPackable>
+    <AspireGeneratedClassesVisibility Condition="'$(AspireGeneratedClassesVisibility)' != 'internal'">public</AspireGeneratedClassesVisibility>
```

Comment:
.props get imported before the .csproj. .targets get imported after.

Setting a property in a `.props` is for cases where we want someone to consume the property later, like in their .csproj. That way someone else can see if it has been set/defaulted yet, and if it hasn't they can change the default.

Something like this should go in the `.targets` (IMO). And then change the condition to be `'$(AspireGeneratedClassesVisibility)' == ''`.